extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -111,7 +111,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     /**
      * @private {JSONType}
      */
-    this.remoteConfig = {};
+    this.remoteConfig_ = {};
```

Comment:
@avimehta in master right now L251 already uses an underscore https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/amp-analytics.js#L251